comparesv.py

```diff
@@ -166,8 +166,8 @@ def fuzzy_column_index(header, headers_list, unmapped_header_indices):
         original_index = unmapped_header_indices[index]
         return original_index
 
-    highest = process.extractOne(header, unmapped_headers)
-    if highest[1] < ROW_THRESHOLD:
+    highest = process.extractOne(header, unmapped_headers, score_cutoff=ROW_THRESHOLD)
+    if highest is None:
```

Comment:
using a score_cutoff allows rapidfuzz to exit early when it is known, that the score_cutoff can not be reached